### PR TITLE
Update version numbers, credits, and changelog for 0.10.4

### PR DESCRIPTION
**Pull request by warp-core**
**Docs and stuff:**

## Details:
- Update version numbers from "0.10.4-alpha" to "0.10.4";
- Update credits (one new contributor), and add some people to the Developers and Editors and Proofreaders lists;
- Update changelog.
- Update the AppStream file.

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+    * Mission changes:
+      * Changed the government of NPC ships in "FW Katya 2B" from "Pirate" to "Bounty Hunter" to better match the lore. (@Anarchist2)
+    * Other:
+      * Tweaked some wording in "FW Pirates: Attack 1". (@hexagonrecursion)
+      * Tweaked some wording in "FW Diplomacy 1C". (@LazerLit)
+      * Random Pug fleets will no longer jump into systems that are not their own. (@Amazinite)
+      * Removed an extraneous item from the "to offer" conditions of "The Book of Skadenga". (@hexagonrecursion)
+  * Game mechanics:
+    * New mechanics:
+      * Outfits can be given the "unique" attribute which will cause hte player to be warned when they are about to be lost. (@TomGoodIdea)
```
Comment (bene-dictator) [inline changelog:30]:
```suggestion
      * Outfits can be given the "unique" attribute which will cause the player to be warned when they are about to be lost. (@TomGoodIdea)
```

File: changelog [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+      * Corrected the offer conditions of, and various other improvements to, "Small Pirate gambling". (@Anarchist2)
+      * Ensure that relevant Quarg governments are friendly to the Kor Mereti after their transformation. (@warp-core)
+      * Use fleets of the "Quarg (Kor Efreti)" government in ruined Korath space, and NPCs with the "Quarg (Hai)" governmment in the "Wanderers: Sestor: Quarg Help 2" and "Wanderers: Sestor: Farpoint Attack 2" missions. (@Saugia)
+      * Cleaned up the descriptions of various spaceport missions. (@Anarchist2)
+      * Removed some duplicated text from "Remnant: Celebration 1". (@warp-core)
+      * Correctly refer to commodity categories in some FW war jobs. (@tibetiroka)
+      * Make the "Quarg (Hai)" government hostile to the "Kor Sestor" government so the Quarg ships will properly aid in the defense of Farpoint. (@qarp-core)
+    * Engine bugs:
+      * Correctly set the application icon on Linux. (@quyykk)
+      * Fix scrolling up with the keyboard from the top of a short list int he load panel causing all the items to snap to the bottom. (@warp-core)
```
Comment (bene-dictator) [inline changelog:16]:
```suggestion
      * Fix scrolling up with the keyboard from the top of a short list in the load panel causing all the items to snap to the bottom. (@warp-core)
```

File: changelog [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+      * Tweaked some wording in "FW Diplomacy 1C". (@LazerLit)
+      * Random Pug fleets will no longer jump into systems that are not their own. (@Amazinite)
+      * Removed an extraneous item from the "to offer" conditions of "The Book of Skadenga". (@hexagonrecursion)
+  * Game mechanics:
+    * New mechanics:
+      * Outfits can be given the "unique" attribute which will cause hte player to be warned when they are about to be lost. (@TomGoodIdea)
+    * Mechanics changes:
+      * Ships are no longer allowed to be outfitted with negative "shield generation" or "hull repair rate" values. (@Arachi-Lover)
+  * User interface:
+    * The "extended jump effects" setting is now off by default, and has an additional intermediate level of motino blur. (@RisingLeaf, @TomGoodIdea)
```
Comment (bene-dictator) [inline changelog:34]:
```suggestion
    * The "extended jump effects" setting is now off by default and has an additional intermediate level of motion blur. (@RisingLeaf, @TomGoodIdea)
```

**Review by bene-dictator — CHANGES_REQUESTED** [review]

**Comment by bene-dictator** [comment]
Just wondering, aren't you missing from the developer list, and me, EjoThims, Terin, and Tibetiroka missing from the proofreaders list?

**Comment by warp-core** [comment]
> Just wondering, aren't you missing from the developer list, and me, EjoThims, Terin, and Tibetiroka missing from the proofreaders list?

I haven't reviewed the criteria used for adding to those lists in the past so I haven't been updating them myself.

File: changelog [hunk 4]
```diff
@@ -1,3 +1,43 @@
+version 0.10.4
```
Comment (TomGoodIdea) [inline changelog:1]:
```suggestion
Version 0.10.4
```

**Pull request merged**